terraform/deployments/govuk-publishing-platform/app_frontend.tf

```diff
@@ -53,12 +97,12 @@ module "draft_frontend" {
     target_group_arn = module.draft_frontend_public_alb.target_group_arn
     container_port   = 80
   }]
-  environment_variables = {} # TODO
-  secrets_from_arns     = {} # TODO
+  environment_variables = local.frontend_defaults.environment_variables
```

Comment:
Interestingly, I can't see how the code before handled the difference in environment variables between frontend and draft-frontend. Perhaps they were just getting the same set of variables?

I think we should probably be overriding some of the env vars here though. At least something like:

```terraform
  environment_variables = merge(
    local.frontend_defaults.environment_variables,
    {
        PLEK_SERVICE_CONTENT_STORE_URI  = local.defaults.draft_content_store_uri
        PLEK_SERVICE_STATIC_URI         = local.defaults.draft_static_uri
    }
  )
```

... maybe it doesn't have to be perfect if it wasn't working before this PR though.